src/dbnode/storage/index/convert/convert_benchmark_test.go

```diff
@@ -155,6 +155,49 @@ func BenchmarkFromSeriesIDAndTags(b *testing.B) {
 	}
 }
 
+func BenchmarkFromSeriesIDAndEncodedTags(b *testing.B) {
+	testData, err := prepareIDAndEncodedTags(b)
+	require.NoError(b, err)
+
+	b.ResetTimer()
+	for i := range testData {
+		_, err := FromSeriesIDAndEncodedTags(testData[i].id, testData[i].encodedTags)
+		require.NoError(b, err)
+	}
+}
+
+func BenchmarkFromSeriesIDAndEncodedTags2(b *testing.B) {
+	testData, err := prepareIDAndEncodedTags(b)
+	require.NoError(b, err)
+
+	b.ResetTimer()
+	for i := range testData {
+		_, err := FromSeriesIDAndEncodedTags2(testData[i].id, testData[i].encodedTags)
+		require.NoError(b, err)
+	}
+}
```

Comment:
Made two implementations, one is more reusable (`FromSeriesIDAndEncodedTags2`), another one has all the code inlined and is slightly faster (`FromSeriesIDAndEncodedTags`). Once agreed which one makes more sense to keep, I will remove the other one.

```
BenchmarkFromSeriesIDAndEncodedTags-12           	 1236018	       979 ns/op
BenchmarkFromSeriesIDAndEncodedTags2-12          	 1118677	      1054 ns/op
```

Other benchmarks for comparison, including one that uses `TagDecoder`:

```
BenchmarkFromSeriesIDAndTagIter-12               	  636643	      1871 ns/op
BenchmarkFromSeriesIDAndTags-12                  	  862752	      1458 ns/op
BenchmarkFromSeriesIDAndTagIter_TagDecoder-12    	  260187	      4554 ns/op
```